### DISCUSSION TITLE
b['irrad_frac_refl_bol@primary'] = 1.7, How to set the value which is greater than 1

### DISCUSSION THREAD

**Discussion by dingxu6207** [phoebe-project/phoebe2#628] [Questions]
b['irrad_frac_refl_bol@primary'] = 1.7

ValueError: value of irrad_frac_refl_bol=1.7 not within limits of [<Quantity 0.>, <Quantity 1.>]

**Comment by kecnry** [comment]
This would mean that the irradiated star irradiates more than the incident flux received from the irradiating star, so probably isn't physical.  But if you really want to force phoebe to do this, you can remove the existing constraint between `irrad_frac_refl_bol` and `irrad_frac_lost_bol`, and then manually override the limits on the parameter:

```
b.run_delayed_constraints()
b.remove_parameters_all(qualifier='irrad_frac_lost_bol')
for param in b.filter(qualifier='irrad_frac_refl_bol').to_list():
    param._limits=[0, None]
```

-Kyle
